.travis.yml

```diff
@@ -8,25 +8,25 @@ os:
     - osx
 
 before_install:
-    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get update && sudo apt-get install -y python-pip zlib1g-dev python-lxml libxml2-dev libxslt1-dev python-dev libboost-dev libboost-python-dev libssh-dev libcurl4-openssl-dev libtool ; fi
+    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get update && sudo apt-get install -y python-pip zlib1g-dev python-lxml libxml2-dev libxslt1-dev python-dev libboost-dev libboost-python-dev libssh-dev libcurl4-openssl-dev libtool python3-lxml; fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew unlink pkg-config && brew install python boost-python libssh jq pkg-config; fi
+    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export PATH=$PATH:~/Library/Python; fi
     - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then docker pull ydkdev/confd-beta; fi
 
 script:
     - export PR=https://api.github.com/repos/$TRAVIS_REPO_SLUG/pulls/$TRAVIS_PULL_REQUEST
     - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo `curl -s $PR | jq -r .head.ref`; fi)
     - export REPO=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_REPO_SLUG; else echo `curl -s $PR | jq -r .head.repo.full_name`; fi)
     - echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, PR=$PR, BRANCH=$BRANCH, REPO=$REPO"
-    - virtualenv myenv
-    - source myenv/bin/activate
-    - pip install -r requirements.txt
-    - python generate.py --profile profiles/test/ydktest.json
-    - pip install gen-api/python/dist/*
-    - export PYTHONPATH=gen-api/python:$PYTHONPATH
-    - python gen-api/python/tests/test_sanity_codec.py
-#    - cd test && sudo ./test_native_ydk.py
-#    - cd ..
-    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then docker run -v $PWD/test/:/root/test/ ydkdev/confd-beta /root/test/travis_ci.sh -r https://github.com/$REPO.git -b $BRANCH ; fi
+    - ./test/osx_linux_tests.sh -g python2.7 -t python2.7
```

Comment:
Also see below. Can we make these arguments more explicit? Instead of g and t, something like "generated_python_version" and "test_python_version"?